src/runtime/pkg/katautils/hook.go

```diff
@@ -136,7 +136,7 @@ func PreStartHooks(ctx context.Context, spec specs.Spec, cid, bundlePath string)
 		return nil
 	}
 
-	return runHooks(ctx, spec, spec.Hooks.Prestart, cid, bundlePath, "pre-start")
+	return runHooks(ctx, spec, spec.Hooks.CreateRuntime, cid, bundlePath, "pre-start")
```

Comment:
s/pre-start/create-runtime/ ?

Reply:
I've swapped it to `createRuntime` #consistency

Reply:
I would suggest that maybe the whole `PreStartHooks` should be removed, but I don't understand the hooks well enough to make that call

Reply:
So the docker tests are failing now - which I think was where I got stuck last time I tried this 😞. I'm not sure whether to just ignore the deprecation warning for now?

Reply:
Just ignore the deprecation warning for now, please.